plugin/loop/README.md

```diff
@@ -80,7 +80,7 @@ requests to itself.
 
 There are many ways to work around this issue, some are listed here:
 
-* Add the following to `kubelet`: `--resolv-conf <path-to-your-real-resolv-conf-file>`.  Your "real"
+* Add the following to your `kubelet` config yaml: `ResolverConfig: <path-to-your-real-resolv-conf-file>` (or set it with the deprecated command flag `--resolv-conf`).  Your "real"
```

Comment:
can you add in which version this was deprecated?

Reply:
done - it was deprecated quite some time ago ... in 1.10.